content/top-level.md

```diff
@@ -47,10 +47,25 @@ The version of the BCO specification used to define the BCO. It is recomended th
 
 ### 2.0.2 BioCompute Object Identifier "object_id"
 
-A unique identifier that should be applied to each BCO instance. These can be assigned by a BCO database engine or manually generated. IDs should never be reused. It is RECOMMENDED that the BCO identifier is based on a [UUID](https://tools.ietf.org/html/rfc4122)s (sometimes called GUIDs) to ensure uniqueness, either as a location-independent URN (e.g. `urn:uuid:2bf8397b-9aa8-47f2-80a7-235653e8e824`) or as part of an identifier permalink, (e.g. `http://repo.example.com/bco/2bf8397b-9aa8-47f2-80a7-235653e8e824`). While the UUID is the preferred method, IDs expressed as a URN or URL are also acceptable.
+A unique identifier that should be applied to each BCO instance. These can be assigned by a BCO database engine or manually generated. IDs should never be reused. It is RECOMMENDED that the BCO identifier is based on a [UUID](https://tools.ietf.org/html/rfc4122)s (sometimes called GUIDs) to ensure uniqueness, either as a location-independent URN (e.g. `urn:uuid:2bf8397b-9aa8-47f2-80a7-235653e8e824`) or as part of an identifier permalink, (e.g. `http://repo.example.com/bco/2bf8397b-9aa8-47f2-80a7-235653e8e824`). While the UUID is the preferred method, IDs expressed as a URN or URL are preferred.
+
+The following is the RECOMMENDED format for a BCO `object_id`, and what has been implementated by the [BCODB](https://github.com/biocompute-objects/bco_api/releases):
+
+The `"object_id"` consists of four different parts: 
+````
+|  <protocol>  |      <hostname>        | <BCO accession>  | <BCO version>  |
+| ------------ | ---------------------- | ---------------- | -------------- |
+|   https://   | biocomputeobject.org/  |   /BCO_000001    |     /1.5       |
+````
+1) Protocol: [see Scheme in RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-3.1)
```

Comment:
`SHOULD` or `MUST` this be HTTPS?

Reply:
I left it ambiguous but maybe `RECOMMENDED` HTTPS?